Dynatrace/dynatrace-operator
# Skipping to send mark for termination event if host not found

## Description

Skipping to send mark for termination event if host not found


## How can this be tested?

1. Create cluster with 2 nodes
2. Edit configMap `dynatrace-node-cache` and change IP address (to simulate host not found from dynatrace) and seen/mark timestamp to past 2h.
3. Delete node from cluster (NOT cordon, delete)

Expected results:
```
{"level":"info","ts":"2023-09-19T15:58:52.195+0200","logger":"nodes","msg":"sending mark for termination event to dynatrace server","dynakube":"dynakube","ip":"10.128.0.223","node":"gke-andrii-test-default-pool-cf2748c9-pt7f"}
{"level":"info","ts":"2023-09-19T15:58:52.734+0200","logger":"dtclient","msg":"hosts cache: ignoring inactive hosts","ids":[""]}
{"level":"info","ts":"2023-09-19T15:58:52.736+0200","logger":"nodes","msg":"skipping to send mark for termination event","dynakube":"dynakube","nodeIP":"10.128.0.223","reason":"host not found","reasonVerbose":"host not found\ngithub.com/Dynatrace/dynatrace-operator/src/dtclient.init\n\t/Users/andrii.soldatenko/work/dynatrace-operator/src/dtclient/config.go:8\nruntime.doInit1\n\t/opt/homebrew/Cellar/go/1.21.1/libexec/src/runtime/proc.go:6740\nruntime.doInit\n\t/opt/homebrew/Cellar/go/1.21.1/libexec/src/runtime/proc.go:6707\nruntime.main\n\t/opt/homebrew/Cellar/go/1.21.1/libexec/src/runtime/proc.go:249\nruntime.goexit\n\t/opt/homebrew/Cellar/go/1.21.1/libexec/src/runtime/asm_arm64.s:1197"}
```

and check that broken node is not exists in configMap

## Checklist
